test/axes/timeAxisTests.ts

```diff
@@ -149,6 +149,54 @@ describe("TimeAxis", () => {
     svg.remove();
   });
 
+  it("tick labels do not overlap with tick marks in top orientation", () => {
+    let svg = TestMethods.generateSVG(400, 100);
+    scale.domain([new Date("2010-01-01"), new Date("2014-01-01")]);
+    axis = new Plottable.Axes.Time(scale, "top");
+
+    function checkTierDisplayPosition (tierDisplayPositions: string[]) {
+      axis.tierLabelPositions(tierDisplayPositions);
+      axis.renderTo(svg);
+      let style = (<any>axis)._element.append("style");
+      style.attr("type", "text/css");
+      style.text(".plottable .axis.time-axis text { font-family: Arial; }");
+      let tickMarks = d3.selectAll(`.${Plottable.Axis.TICK_MARK_CLASS}:not(.${Plottable.Axis.END_TICK_MARK_CLASS})`);
+      assert.operator(tickMarks.size(), ">=", 1, "There are at least one tick marks in the test");
+      let tickLabels = d3.selectAll(`.${Plottable.Axis.TICK_LABEL_CLASS}`)
+      .filter(function(d, i){
+        return window.getComputedStyle(this).visibility !== "hidden";
```

Comment:
`d3.style` also does not work on IE9. As the other test in this file use `window.getComputedStyle`, I might still want to use `window.getComputedStyle`.